tests/VersionsCheckTest.php

```diff
@@ -125,14 +126,15 @@ public function testMultiplePackagesComparison(array $packagesData, $preferStabl
 
         $this->checkPackages();
 
-        $this->assertSame(sprintf(<<<EOF
-<warning>Some packages are not up to date:</warning>
+        $expectedContent = str_replace('/', '\\/', quotemeta(implode("\n", $shouldBeUpdatedOutput)));
+        $this->assertRegExp(sprintf(<<<EOF
+/^<warning>%d packages are not up to date:<\/warning>
```

Comment:
Why keeping regexp for this? Single string comparison with a `sprintf` would be enough.